### ISSUE TITLE
pm2: command not found

### ISSUE THREAD

**Issue by dec1** [neurochain/prototype#1]
```
 ./scripts/nodesLauncher.sh
1 12345 5000
./scripts/nodesLauncher.sh: line 21: pm2: command not found
number of nodes = 2
```

 _Also small problem " ./scripts/nodesLauncher.sh" should executable (when you check it out of repo)_

**Comment by dec1** [comment]
presumably theres a way not to force the user to globally install things like pm2?

**Comment by neurochain** [comment]
Hello, 
sorry for the delay !
This problem will be corrected soon ! I'm on it !
you are right, pm2 should not be mandatory
Thank you for your interest

**Comment by dsir-minute** [comment]
hi,
execution flags do get lost due to github, this is a well known issue, you then just need to mention how to retrieve all those lost permissions after a git clone.
Anyway, I quickly fixed the permission isuue on the shell script, but got the same pm2 issue as @dec1 ,
Pay more attention guys, please!
tks!

**Comment by dsir-minute** [comment]
what is the purpose of this pm2 ? 
edit : cf http://pm2.keymetrics.io/

**Comment by dsir-minute** [comment]
ok, 
sudo npm install pm2 -g
solved the issue,
then everything apparently went without any further error

**Comment by dec1** [comment]
yes, but I dont want to globally (-g) alter my local node installation just to run your script.
Cant this be solved without a global installation.? ie into same dir so when I delete dir its definitely gone then

**Comment by dsir-minute** [comment]
both 
npm install pm2
and
npm install pm2 -g
failed, thus my
sudo npm install pm2 -g
I guess pm2 can be uninstalled after we finish testing neurochain

**Comment by neurochain** [comment]
We are working an a new version easier to run our prototype.
This new version will be released tomorrow.

**Comment by dec1** [comment]
ok thanks
Maybe a few lines of docu describing what it does (if its not obvious from output) would be a nice addition too :-)
